test/azure/legacy/Expected/AcceptanceTests/AzureBodyDuration/generated_samples/duration_get_null.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from bodyduration import AutoRestDurationTestService
+
+"""
+# PREREQUISITES
+    pip install autorestdurationtestservice
+# USAGE
+    python duration_get_null.py
+"""
+
+
+def main():
+    client = AutoRestDurationTestService()
+
+    response = client.duration.get_null()
```

Comment:
To make the samples can be picked up at doc site, the "--include-x-ms-examples-original-file" should be enabled as modelerfour option, and the "x-ms-original-file" of the example should be shown as comment in the generated sample file.